malaria-ios/Managers/UserSettingsManager.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+class UserSettingsManager{
+    
+    /// Sets an UserSetting
+    /// :param: `UserSetting`: the setting to be changed
+    /// :param: `Bool`: value
+    class func setBool(key: UserSetting, _ value: Bool){
```

Comment:
Yes, it makes! However I would need to create a `get`and a `set`for each user setting. However, since there is only that setting for now, I will follow your first suggestion.